cmd/jag/commands/scan.go

```diff
@@ -124,7 +124,6 @@ looping:
 		if err != nil {
 			fmt.Println("Failed to parse identify", err)
 		} else if dev != nil {
-			// TODO (jesper): Once names are real unique, use name in map.
 			devices[dev.Address] = *dev
```

Comment:
Use dev.ID here?

Reply:
It is probably better to just use the address. After all, this is how we found the devices.